src/utils/switch-site.ts

```diff
@@ -18,10 +18,16 @@ const SITE_TO_EMAIL_MAP: Record<SiteKey, string> = {
 export const setSiteCookie = (site: SiteKey) =>
   Cookies.set(COOKIE_USER_KEY, SITE_TO_EMAIL_MAP[site])
 
-/** Set the initial site if the demo is visited for the first time. */
-export function setInitialSiteCookie() {
-  if (!Cookies.get(COOKIE_USER_KEY)) setSiteCookie(DEFAULT_SITE)
-}
+export const getCurrentSite = (): SiteKey =>
+  JSON.parse(localStorage.getItem(SITE_KEY)!) || DEFAULT_SITE
+
+/**
+ * Sync the user cookie for the API with the site in localStorage.
+ *
+ * Uses the site from localStorage as the initial user cookie.
+ * This is a migration for people who have previously visited the demo.
+ **/
+export const syncInitialSiteCookie = () => setSiteCookie(getCurrentSite())
```

Comment:
This is from https://github.com/metabase/shoppy/pull/69

Reply:
Removed as of https://github.com/metabase/shoppy/pull/71/commits/4e688df9448b307b4b8c1f3a495026e3d9bf4ed6 as we use query parameters instead of cookies for cacheability of `/products` and `/categories` endpoint